bots/images/scripts/lib/atomic.setup

```diff
@@ -53,6 +53,17 @@ else
     checkout=$(atomic host status --json | python -c 'import json; import sys; j = json.loads(sys.stdin.readline()); print [x for x in j["deployments"] if x["booted"]][0]["checksum"]')
 fi
 
+# Download the libssh RPM which we'll use for package ovelay
+# The only way to do this is via a container
+. /etc/os-release
+REPO="updates"
+if [ "$ID" = "rhel" ]; then
```

Comment:
Really? The docker run command below the if statement is what does the download of libssh RPM.